src/test/scala/org/kidsfirstdrc/dwh/variantDb/json/VariantsToJsonJobSpec.scala

```diff
@@ -59,12 +60,38 @@ class VariantDbJsonSpec extends AnyFlatSpec with GivenWhenThen with WithSparkSes
 
   "VariantDbJson" should "transform data to the right format" in {
 
-    val result = VariantDbJson.transform(data)
-
-    result.show(false)
+    val result = new VariantsToJsonJob(realeaseId).transform(data)
+    val parsedResult = result.as[VariantToJsonJobModel.Output].collect()
+    val `1k_genomes`: Frequency =
+      result.select(
+        "frequencies.1k_genomes.an",
+        "frequencies.1k_genomes.ac",
+        "frequencies.1k_genomes.af",
+        "frequencies.1k_genomes.homozygotes",
+        "frequencies.1k_genomes.heterozygotes")
+      .as[VariantToJsonJobModel.Frequency].collect().head
+    val variant = parsedResult.head
 
     result.columns should contain allOf
-      ("chromosome", "start", "end", "reference", "alternate", "studies",  "frequencies", "clinvar", "dbsnp_id", "consequences")
+      ("chromosome", "start", "end", "reference", "alternate", "studies",
+        "frequencies", "clinvar", "dbsnp_id", "release_id", "consequences")
+
+    //1. make sure we have only 1 row in the result
+    parsedResult.length shouldBe 1
+    //2. data validation of that row
+    variant.studies.size shouldBe 3
```

Comment:
I think you cloud use a shouldBe with an instacne of the case class instead of comparing each fields